planning/motion_velocity_planner/autoware_motion_velocity_obstacle_velocity_limiter_module/benchmarks/collision_checker_benchmark.cpp

```diff
@@ -72,12 +72,8 @@ int main()
       CollisionChecker naive_collision_checker(obstacles, nb_points + 1, nb_lines * 100);
       const auto naive_constr_end = std::chrono::system_clock::now();
       const auto rtt_check_start = std::chrono::system_clock::now();
-      for (const auto & polygon : polygons)
-        const auto rtree_result = rtree_collision_checker.intersections(polygon);
       const auto rtt_check_end = std::chrono::system_clock::now();
       const auto naive_check_start = std::chrono::system_clock::now();
-      for (const auto & polygon : polygons)
-        const auto naive_result = naive_collision_checker.intersections(polygon);
```

Comment:
This code is to measure runtimes so we do not want to remove these lines.

Reply:
Thank you for your advice.

I have made the change to suppress the warning by commenting.